src/components/Common/Typography/Text.tsx

```diff
@@ -22,6 +22,7 @@ interface Props {
   style?: FontStyle
   as?: 'span'
   title?: string
+  uppercase?: boolean
```

Comment:
Maybe I'd change this to "transform", and it can receive `transform="uppercase"`, so if in the future we want to add capitalize, for example, it'd be easier.